client/views/admin/apps/AppDetailsPageContent.tsx

```diff
@@ -46,26 +46,26 @@ const AppDetailsPageContent: FC<AppDetailsPageContentProps> = ({ data }) => {
 					</Box>
 					<Box display='flex' flexDirection='row' alignItems='center' justifyContent='space-between'>
 						<Box flexGrow={1} display='flex' flexDirection='row' alignItems='center' marginInline='neg-x8'>
-							<AppStatus app={data} marginInline='x8' />
+							<AppStatus app={app} marginInline='x8' />
 							{!installed && (
 								<PriceDisplay purchaseType={purchaseType} pricingPlans={pricingPlans} price={price} showType={false} marginInline='x8' />
 							)}
 						</Box>
-						{installed && <AppMenu app={data} />}
+						{installed && <AppMenu app={app} />}
 					</Box>
 				</Box>
 			</Box>
 			<Divider />
 
-			{data.licenseValidation && (
+			{app.licenseValidation && (
```

Comment:
from line 60 to line 70 can be improved by breaking the validations into small functions to make easy the read around this code.